docs/DeveloperGuide.md

```diff
@@ -236,27 +236,55 @@ _{Explain here how the data archiving feature will be implemented}_
+| **       | Intermediate user | wants to manage my ingredients                                | easily find the ingredients i want                                     | quickly find ingredients and save time on home cooking                                |
+|          | Intermediate user | wants to be healthier                                         | create my diet plan                                                    | better stick to a healthy diet.                                                       |
+|          | Intermediate user | keen on following a certain diet                              | keep track of my recipes for that specific diet                        | better stick to my diet plan                                                          |
+|          | Intermediate user | organise my diet plan                                         | remove recipes that do not belong in a certain diet plan               | have a neat and accurate diet plan                                                    |
+|          | Intermediate user | want to track my diet progress                                | want to view my calorie and food intake statistics                     | I can review my diet and work on being even healthier                                 |
+| *        | Expert user       | is forgetful                                                  | have a table of commands and shortcuts                                 | I can use the app more effieciently.                                                  |
+|          | Expert user       | is organized                                                  | group recipes into labelled sections by adding tags                    | I can easily find recipes of the same section                                         |
+|          | Expert user       | is neat                                                       | remove recipes from labelled sections                                  | each section is not cluttered with recipes that don't belong in the section           |
+|          | Expert user       | like to browse recipe                                         | filter the recipe base by my tags                                      | I can browse the recipe easily                                                        |
+|          | Expert user       | likes convenience                                             | be able to export the recipes in the app to another device             | I can refer to the data from another device and share it with others                  |
+|          | Expert user       | is a developer                                                | customize the app to my own preferences                                | the app is open-source allowing me to contribute and extend its further               |
```

Comment:
Same with previous, should change the "the app is open source..." to fit the "so that I can..." form?

Reply:
Thanks for pointing it out, made the changes